### PR TITLE
Dockerize deamons with docker compose

### PR DESCRIPTION
**Pull request by napoly**
~all~ credit here goes to @erciccione for #297 .. should solve #284 (would be a shame to leave this work behind..)

@abouolia  `No daemon connection` for arbitrator should be fixed here.

### CODE REVIEW

File: docker/monero/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+FROM ubuntu:20.04 as downloader
+
+RUN set -ex && \
+    apt update && \
+    apt --no-install-recommends --yes install \
+        wget \
+        ca-certificates
+
+# We use the existent haveno_deps.sh script to download and verify the Monero binaries
+RUN set -ex && wget https://raw.githubusercontent.com/napoly/haveno/docker/scripts/haveno_deps.sh && \
```
Comment (napoly) [inline docker/monero/Dockerfile:12]: should be changed to https://raw.githubusercontent.com/haveno-dex/haveno/master/scripts/haveno_deps.sh before merging to master

File: docker/haveno/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        make \
+        git
+
+RUN set -ex && adduser --system --group --disabled-password haveno && \
+    mkdir -p /home/haveno && \
+    chown -R haveno:haveno /home/haveno
+
+USER haveno
+
+RUN set -ex && git clone -b docker https://github.com/napoly/haveno.git /home/haveno/haveno
```
Comment (napoly) [inline docker/haveno/Dockerfile:17]: change to:
`RUN set -ex && git clone https://github.com/haveno-dex/haveno.git /home/haveno/haveno`
before merging to master

**Comment by woodser** [comment]
Please rebase.

File: docs/installing.md [hunk 3]
```diff
@@ -68,6 +68,8 @@ If you don't use *screen*, open 4 terminal windows and run in each one of them:
   1. `make seednode-local`
   2. `make user1-desktop-local` or if you want to run user1 as a daemon: `make user1-daemon-local`
   3. `make user2-desktop-local` or if you want to run user2 as a daemon: `make user2-daemon-local`
+  4. `make arbitrator-desktop-local` or if you want to run arbitrator as a daemon: `make arbitrator-daemon-local`
+  5. Run a local price node [../haveno-pricenode/blob/main/README.md](https://github.com/haveno-dex/haveno-pricenode/blob/main/README.md)
```
Comment (woodser) [inline docs/installing.md:72]: Recommended change for simplicity:

Optionally [run a local price node](https://github.com/haveno-dex/haveno-pricenode/blob/main/README.md).

**Comment by napoly** [comment]
partially dismantling this PR since a full fledged solution can be found at haveno-ts [#150](https://github.com/haveno-dex/haveno-ts/pull/150)

**Pull request merged**